ex02.exs

```diff
@@ -79,26 +78,25 @@ end
 ########################################
 
 defmodule Ex02 do
+  use Agent
 
   def new_counter(initial_value \\ 0) do
-    # « your code goes here»
+    {:ok, counter} = Agent.start_link(fn -> initial_value end)
+    counter
   end
 
   def next_value(agent) do
-    # « your code goes here»
+    value = Agent.get_and_update(agent, fn state -> {state, state + 1} end)
```

Comment:
# -1

There's a (correct) warning that `value` is unused. 

In general, you should always fix warning in code—treat them as errors. That's for everything, not just my assignments.